target_chains/ethereum/contracts/contracts/pyth/Pyth.sol

```diff
@@ -71,16 +71,15 @@ abstract contract Pyth is
     function updatePriceFeeds(
         bytes[] calldata updateData
     ) public payable override {
-        // TODO: Is this fee model still good for accumulator?
-        uint requiredFee = getUpdateFee(updateData);
-        if (msg.value < requiredFee) revert PythErrors.InsufficientFee();
-
+        uint8 totalNumUpdates = 0;
```

Comment:
@ali-bahjati is `uint8` enough for holding the total number of updates we could get in one call?

Reply:
yeah i'm not sure we can fit more gas-wise (the block gas limit). That's why on wire format it's u8. Do you think people will need more?

Reply:
this would be for summing up all the individual `numUpdates` so depends on how big `updateData` could potentially be. I think for now 255 would be enough but we could possibly need more in the future?

Reply:
oh ok ok. I might just change it to uint. Since word-size in evm is 32 bytes it shouldn't change much gas-wise.